lxl-web/src/app.css

```diff
@@ -49,4 +49,22 @@
 	.container-fluid {
 		@apply mx-auto px-4 sm:px-8;
 	}
+
+	.button-primary {
+		@apply flex h-10 items-center justify-center gap-2 rounded-md px-4  py-2 text-primary-inv text-4-cond-bold gradient-primary hover:gradient-secondary;
+		box-shadow:
+			0px 1px 0px 0px rgba(82, 51, 20, 0.16),
+			0px 1px 0px 0px,
+			rgba(239, 214, 189, 0.24) inset;
+	}
+	.button-ghost {
+		@apply flex h-10 items-center justify-center gap-2 rounded-md px-4 py-2 text-primary text-4-cond-bold;
```

Comment:
I think the text style used for buttons in the design is `text-3-cond-bold`, which is reasonable, it looks a bit too big now.

Reply:
Also the dots (...)
should probably have the same text style and maybe aligned to the bottom (`items-end`)?

Reply:
Other than that, lookin' good! :)

See if you can resolve the comments we made, then we'll fix the merge conflict together.

Reply:
Thank you I will try to resolve this :D